README.md

```diff
@@ -30,6 +30,7 @@ in protocol buffer or text format and dumps the result as JSON to
 
 Installing and building:
 
+    $ cd cmd/prom2json 
```

Comment:
I thought it is, because the README is at the repository-root, while the building is done within the folder `cmd/prom2json`.